src/3045R0_quantities_and_units_library.md

```diff
@@ -757,13 +758,13 @@ For example:
 
 - _length_ ($L$), _mass_ ($M$), _time_ ($T$), _electric current_ ($I$), _thermodynamic temperature_ ($Θ$),
   _amount of substance_ ($N$), and _luminous intensity_ ($J$) are the base dimensions of the ISQ.
-- A derived dimension of _force_ in the ISQ is denoted by $dim F = LMT^{–2}$.
-- IEC 80000 provides _traffic intensity_ quantity that is measured in Erlangs but not in a unit one,
+- A derived dimension of _force_ in the ISQ is denoted by $dim F = LMT^{–2}$. <!-- The space in "dim F" isn't rendered. -->
```

Comment:
This might work:
$\text{dim }F = LMT^{–2}$
```suggestion
- A derived dimension of _force_ in the ISQ is denoted by $\text{dim }F = LMT^{–2}$. <!-- The space in "dim F" isn't rendered. -->
```